api/control/v1alpha1/types.go

```diff
@@ -153,7 +153,19 @@ type DownsyncPolicyClause struct {
 	// +optional
 	CreateOnly bool `json:"createOnly,omitempty"`
 
-	// statusCollectors is a list of StatusCollectors name references that are applied to the selected objects.
+	// `statusCollection` holds the rules for collecting status from the WECs.
+	// +optional
+	StatusCollection *StatusCollection `json:"statusCollection,omitempty"`
```

Comment:
Does a non-zero default value for StalenessThresholdSecs play nice with "omitempty" and "+optional"?

Reply:
Yes.

Reply:
As far as I am concerned, it would be OK to say that the default is zero and zero means "unspecified" and when no threshold is specified then a threshold of 120 seconds is used. Another way to handle "unspecified" is to make StalenessThresholdSecs a _pointer to_ an integer (so that `nil` can represent "unspecified").

Reply:
I think it's fine and simple as it is currently: if StatusCollection is set, staleness defaults to 120. If it is not set, staleness is not included. Do you agree with keeping it as it is?

Reply:
I am not convinced, but I think it is moot since ManifestWork does not support the useful notion of "stale".